setup.py

```diff
@@ -39,7 +39,7 @@
       scripts=glob('scripts/*py'),
       setup_requires=['numpy >= 1.9.2'],
       install_requires=[
-          'IPython < 4.0.0',
+          'IPython',
```

Comment:
That means it isn't working: if it is working, you should not notice a difference when operating on a .md or .ipynb as the conversion is implicit